application/electron/src/controllers/electron/controller.electron.menu.ts

```diff
@@ -21,6 +21,10 @@ import * as os from 'os';
 const MAX_NUMBER_OF_RECENT_FILES = 20;
 
 const MENU_TEMPLATE = [
+    {
+        label: 'Chipmunk',
+        submenu: [ ],
+    },
```

Comment:
This seems wrong. The existing code below already adds this if platform==darwin.